docs/installing.md

```diff
@@ -0,0 +1,61 @@
+# Installing
+
+In most cases all you need to do is
+```bash
+pip install ipympl
+```
+or alternatively with `mamba`/`conda`
+
+```bash
+mamba install -c conda-forge ipympl
+```
+
+and then restart Jupyter.
+
+
+### Jupyterlab < 3
+If you use JupyterLab 2, you still need to install the labextension manually:
+
+```bash
+conda install -c conda-forge nodejs
+jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-matplotlib
+```
+
+
+<!--
+## Sagemath CoCalc
+TODO
+-->
```

Comment:
Anyone have an idea if/how to get ipympl working in sagemath/cocalc?

Reply:
Now that there's less noise here: @williamstein I couldn't find an obvious way to do this after some brief searching and the one attempt I had didn't work. Would love to include instructions though